### ISSUE TITLE
Pole Kriteria

### ISSUE THREAD

**Issue by vrana** [HlidacStatu/verejne-zakazky#7]
Ve výsledné struktuře je pole `Kriteria`. Ve zdrojovém XML jsem našel jen `zakazka/VZ/zpusob_hodnoceni`, což je jeden enum z těchto hodnot:

- Nejnižší nabídková cena
- Ekonomická výhodnost nabídky
- Přímé zadání (zakázka byla přímo zadána danému dodavateli)
- Oslovení několika zájemců (více než jeden zájemce byl vyzván k podání nabídky)
- Veřejná soutěž (výzva byla veřejně oznámena, s možností účasti předem neomezeného počtu dodavatelů)

Popis hodnocení textem je v `zakazka/VZ/cast_zakazky/zpusob_hodnoceni_textem`, ale pole `cast_zakazky` bývá obvykle prázdné.

Hlavní rozdíl vidím v tom, že `Kriteria` je seznam několika kritérií, ale v XML je kritérium jen jedno.

**Comment by michalblaha** [comment]
Ano, jedno kriterium z profilu premapujeme  do pole Kriteria z transformovaneho JSON.
